exporter/otelarrowexporter/README.md

```diff
@@ -264,44 +264,107 @@ exporters:
 
 ### Batching Configuration
 
-This exporter includes a new, experimental `batcher` configuration for
-batching in the `exporterhelper` module, but this mode is disabled by
-default.  This batching support works when combined with
-`queue_sender` functionality.
+### Option 1: Batching with back-pressure
 
+To configure an OpenTelemetry Collector pipeline for both batching and
+back-pressure, use of a custom component, the Concurrent Batch Processor,
+available in the OTel-Arrow project repository, is required. We have not
+included this in the Collector-Contrib repository because equivalent
+functionality is being added as a standard exporter-batcher mechanism and the
+new exporter-batcher functionality is still experimental.
```

Comment:
As a user, how would I configure this to be include this into my collector?

Would it be possible add like a otb (collector builder) example? Or a link earlier in this text to go follow this and learn more there?

My concern that this is a lot of "expert" mode configuration with no warning that it is.

Reply:
This is a response to the reality today, which is that the none of the built-in support actually helps with the problem I am trying to solve--which is to have a synchronous pipeline with error transmission that is not limited to one export at a time. I am desperate to fix these problems! See the issue:

https://github.com/open-telemetry/opentelemetry-collector/issues/11308

If you feel that this "expert-mode" is really a problem, then we should address the underlying issue--I ask you to approve https://github.com/open-telemetry/opentelemetry-collector/pull/11324. If you do not feel that there is a problem, then let's merge this PR.